cmd/dpiprobe/interfaceutils.go

```diff
@@ -27,31 +28,31 @@ func FindPcapInterfaceName(ipAddress net.IP) (string, error) {
 	return "", nil
 }
 
-func findOutgoingPcapInterfaceNameAndIp(targetIp *net.IPAddr) (string, *net.IPAddr, error) {
-	initialConn, err := net.DialUDP("udp", nil, &net.UDPAddr{IP: targetIp.IP, Port: 443})
+// findOutgoingPcapInterfaceNameAndIp Finds outgoing interface name and IP for packet capture
+func findOutgoingPcapInterfaceNameAndIP(targetIP *net.IPAddr) (string, *net.IPAddr, error) {
+	initialConn, err := net.DialUDP("udp", nil, &net.UDPAddr{IP: targetIP.IP, Port: 443})
 	if err != nil {
 		return "", nil, err
 	}
 
-	localInterfaceIp := initialConn.LocalAddr().(*net.UDPAddr).IP
+	localInterfaceIP := initialConn.LocalAddr().(*net.UDPAddr).IP
```

Comment:
Why did you rename Ip to IP everywhere ? This change in unnecessary and unrelated to this PR. Besides, I like it as Ip :-)